zb-classifier.js

```diff
@@ -1088,6 +1195,14 @@ class ZigbeeClassifier {
     }
   }
 
+  initMultiLevelButton(node, genLevelCtrlOutputEndpoint) {
+    node.name = `${node.id}-button`;
+    node.type = 'multiLevelButton';
+    node['@type'] = ['MultiLevelButton'];
```

Comment:
If you change the other property to `PushedProperty`, you could add `PushButton` here.

Reply:
But it isn't really a push button. There are 2 buttons. You push one of them to turn it on, and you push the other one to turn it off. You push and hold the on button to increase the level and push and hold the off button to decrease the level.

So it feels quite different from a push button to me.